### PR TITLE
248 & 249: Privacy Policy page and Accessibility Statement page

### PR DESCRIPTION
**Pull request by jamesgriff**

### CODE REVIEW

File: grc/__init__.py [hunk 1]
```diff
@@ -89,6 +89,10 @@ def make_before_request():
     app.register_blueprint(submitAndPay)
     app.add_url_rule('/submit-and-pay', endpoint='index')
 
+    # Submit and pay
```
Comment (alistairknight) [inline grc/__init__.py:92]: Can you change the comment to "Policies"

File: grc/policies/__init__.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from flask import Blueprint, render_template
+
+policies = Blueprint('policies', __name__)
+
+
+@policies.route('/privacy-policy', methods=['GET', 'POST'])
```
Comment (alistairknight) [inline grc/policies/__init__.py:6]: Don't need POST for either of these

File: grc/__init__.py (hunk 1 above)
Comment (jamesgriff) [reply grc/__init__.py:92]: Good spot - thanks :-)

File: grc/policies/__init__.py (hunk 2 above)
Comment (jamesgriff) [reply grc/policies/__init__.py:6]: Good spot - thanks :-)

**Pull request merged**